nvm.sh

```diff
@@ -1303,6 +1320,30 @@ nvm_die_on_prefix() {
   fi
 }
 
+# Succeeds if $IOJS_VERSION represents an io.js version that has a
+# Solaris binary, fails otherwise.
+# Currently, only io.js 3.3.1 has a Solaris binary available, and it's the
+# latest io.js version available. The expectation is that any potential io.js
+# version later than v3.3.1 will also have Solaris binaries.
+iojs_version_has_solaris_binary() {
+  local IOJS_VERSION=$1
+  nvm_version_greater_than_or_equal_to "$(nvm_strip_iojs_prefix $IOJS_VERSION)" v3.3.1
+}
+
+node_version_has_solaris_binary() {
+  local NODE_VERSION=$1
+  nvm_version_greater_than_or_equal_to "$NODE_VERSION" v0.8.6
+}
+
+nvm_has_solaris_binary() {
```

Comment:
Please also add unit tests for `nvm_has_solaris_binary` and `node_version_has_solaris_binary`